enos/enos.vars.hcl

```diff
@@ -47,14 +51,24 @@
 // of the build workflow.
 // crt_bundle_path = "./boundary_linux_amd64.zip"
 
-// The port the ALB will listen on to proxy controller API requests. This defaults
-// to 9200
-// alb_listener_api_port = 9200
+// Number of controller instances to create. Applies to AWS scenarios only.
+// controller_count = 1
 
-// Generally, if there's failure in the test suite for any reason, enos/terraform will throw an error and you
-// would not be able to access the environment variables needed to test locally. Enabling this
-// will ensure that the enos scenario passes.
-// e2e_debug_no_run = true
+// Number of worker instances to create. Applies to AWS scenarios only.
+// worker_count = 1
+
+// Number of target instances to create. Applies to AWS scenarios only.
+// target_count = 1
 
-// Timeout for `go test` execution in the e2e tests, 10m default
-// go_test_timeout = "10m"
+// The directory that contains the copy of the boundary cli that the e2e tests
+// will use in CI. Only needed if e2e_debug_no_run = false.
+// local_boundary_dir = "/Users/<user>/.go/bin"
+
+// The directory that contains the source code of boundary/boundary-enterprise.
+// This is used in docker scenarios in CI in order to mount the source code into
```

Comment:
Is this only need when running in docker? or is need always?

Reply:
1. I put this in the `Optional Variables` section
2. It's only needed if `e2e_debug_no_run = false` (where enos would run the tests automatically). This variable is specifically for the backend tests. `local_boundary_ui_src_dir` is a similar thing, but for the Admin UI tests.

Reply:
No, all good, yeah I notice was on Optional variables, and now I realize the `e2e_debug_no_run`. I think is clear enough, I wanted to double check 😃